multi-ai-cli/core/cache.py

```
@@ -1,58 +1,23 @@
-import json, os, re, sys
+import json, os, sys
 from pathlib import Path
 from typing import List, Dict, Optional
```

Ruff (0.16.2)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)